modules/avatar/avatar.go

```diff
@@ -22,8 +23,9 @@ import (
 	_ "golang.org/x/image/webp" // for processing webp images
 )
 
-// AvatarSize returns avatar's size
-const AvatarSize = 290
+// DefaultAvatarSize is used for avatar generation, usually the avatar image saved in server won't be larger than this value.
+// Unless the original file is smaller than the resized image.
+const DefaultAvatarSize = 256
```

Comment:
This needs to take into account `Avatar.RenderedSizeFactor` to allow High DPI rendering, so in the default config 256*3 = 768.

Reply:
Done